TwitchPlaysAssembly/Src/Commands/GameCommands.cs

```diff
@@ -630,12 +641,22 @@ public static void CallAllQueuedCommands(string user, bool isWhisper)
 		var allCommands = TwitchGame.Instance.CommandQueue.ToList();
 		TwitchGame.Instance.CommandQueue.Clear();
 		TwitchGame.ModuleCameras?.SetNotes();
+		List<IRCMessage> cmdsToExecute = new List<IRCMessage>();
 		foreach (var call in allCommands)
 		{
 			IRCConnection.SendMessageFormat("Calling {0}: {1}", call.Message.UserNickName, call.Message.Text);
-			IRCConnection.ReceiveMessage(call.Message);
+			if (TwitchGame.Instance.Bombs.Any(x => x.BackdoorHandleHack))
+				IRCConnection.SendMessage("Hack detected, waiting until the hack is over to execute this command.");
```

Comment:
Why is this being done in the foreach? Wouldn't that mean it would send a message for every command? I think it would be better to move this message to the WaitForCall method.